FinnHub.Core/FinnHubException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace FinnHub.Core
+{
+    public class FinnHubException : Exception
+    {
+        public int StatusCode { get; set; }
+
+        public string Content { get; set; }
+    }
```

Comment:
Needs constructors

[How to create user-defined exceptions](https://docs.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions)

Reply:
Exception definition changed to use the constructor instead of properties.